python/pyarrow/tests/test_fs.py

```diff
@@ -1463,6 +1463,16 @@ def test_azurefs_options(pickle_module):
     assert pickle_module.loads(pickle_module.dumps(fs3)) == fs3
     assert fs3 != fs2
 
+    fs4 = AzureFileSystem(account_name='fake-account-name',
+                          sas_token='fakesastoken')
+    assert isinstance(fs4, AzureFileSystem)
+    assert pickle_module.loads(pickle_module.dumps(fs4)) == fs4
+    assert fs4 != fs3
+    
+    with pytest.raises(ValueError):
+        AzureFileSystem(account_name='fake-account-name', account_key='fakeaccount',
+                        sas_token = 'fakesastoken')
```

Comment:
```suggestion
                        sas_token='fakesastoken')
```

Reply:
Ran the linter 5 minutes after this comment. Sorry! All set.

Reply:
Could you check https://github.com/apache/arrow/pull/45706#discussion_r1988211643 too?